paddle/fluid/framework/ir/mkldnn/cpu_quantize_pass.cc

```diff
@@ -274,10 +282,16 @@ void CPUQuantizePass::QuantizeConv(Graph* graph,
 
     conv_op->Op()->SetAttr("Scale_weights", filter_scale);
 
-    bool is_output_unsigned{false};
-    auto output_scale = GetScaleValueForNode(conv_output, &is_output_unsigned);
-    DequantizeOutput(g, conv_op, conv_output, "Output", output_scale,
-                     is_output_unsigned, "Scale_out");
+    // if quantization scale is missing for output tensor, return fp32 data
+    if (AreScalesPresentForNodes(conv_op, {conv_output})) {
+      bool is_output_unsigned{false};
+      auto output_scale =
+          GetScaleValueForNode(conv_output, &is_output_unsigned);
+      DequantizeOutput(g, conv_op, conv_output, "Output", output_scale,
+                       is_output_unsigned, "Scale_out");
+    } else {
+      conv_op->Op()->SetAttr("force_fp32_output", true);
```

Comment:
Doesn't this imply that we could get rid of `x + dequant -> x_with_force_fp32_output` passes?

Reply:
For now, yes. However, I would not remove the pass at the moment. It could be removed in a separate PR.